pandas/core/generic.py

```diff
@@ -3751,6 +3752,66 @@ def last(self, offset):
         start = self.index.searchsorted(start_date, side='right')
         return self.ix[start:]
 
+    def rank(self, axis=0, method='average', numeric_only=None,
+             na_option='keep', ascending=True, pct=False):
+        """
+        Compute numerical data ranks (1 through n) along axis. Equal values are
+        assigned a rank that is the average of the ranks of those values
+
+        Parameters
+        ----------
+        axis: {index (0), columns (1)), default 0
```

Comment:
can you use the same type description as before in the dataframe docstring? (`{0 or 'index', 1 or 'columns'}, default 0`) -> quotes to indicate it is a string argument